spec/models/vm/retirement_management_spec.rb

```diff
@@ -143,6 +143,18 @@
     @vm.raise_retirement_event(event_name)
   end
 
+  it "#raise_retirement_event with current user" do
+    User.current_user = FactoryGirl.create(:user_with_group, :userid => 'freddy')
```

Comment:
@lfu Instead of directly setting the User.current user you should use the User.with_user e.g. https://github.com/ManageIQ/manageiq/blob/master/spec/models/miq_event_spec.rb#L112